gf-webapp/src/component/filter/RoomsBlockWithFilters.tsx

```diff
@@ -0,0 +1,99 @@
+import {useEffect, useState} from "react";
+import {Room} from "src/model/Room/Room";
+import {RoomItem} from "src/logic/rooms/roomsPage/roomBlock/roomItem/RoomItem";
+import {RoomApiService} from "src/service/RoomApi/RoomApi";
+import {useFilterRooms} from "src/component/filter/FilterContext";
+
+export const RoomsBlockWithFilters = () => {
+  const {searchValue, adultsValue, childrenValue, dateInValue, dateOutValue} = useFilterRooms();
+
+  const [rooms, setRooms] = useState<Room[] | null>(null);
+
+  const initRooms = async () => {
+    const roomsList = await RoomApiService.getAllRooms();
+    setRooms(roomsList);
+  };
+
+  useEffect(() => {
+    async function onRoomsInitialized() {
+      await initRooms();
+    }
+    onRoomsInitialized();
+  }, []);
+
+  const adultsFilterCondition = (roomFiltering: Room) => {
```

Comment:
What mean room filtering? Do u mean filtered?

Reply:
Change to room